README.md

```diff
@@ -1,247 +1,260 @@
+[https://centrifugal.github.io/centrifugo/server/http_api/#presence](https://centrifugal.github.io/centrifugo/server/http_api/#presence)
 
 ```ruby
-client.generate_channel_sign(params[:client], params[:channels], "{"name": "John"}")
-```
-
-On server side using rails, you can authenticate private channels like so:
+client.presence(channel: 'chat') 
+
+# {
+#   'result' => {
+#     'presence' => {
+#       'c54313b2-0442-499a-a70c-051f8588020f' => {
+#         'client' => 'c54313b2-0442-499a-a70c-051f8588020f',
+#         'user' => '42'
+#       },
+#       'adad13b1-0442-499a-a70c-051f858802da' => {
+#         'client' => 'adad13b1-0442-499a-a70c-051f858802da',
+#         'user' => '42'
+#       }
+#     }
+#   }
+# }
+``` 
+
+#### Presence stats
+
+Get short channel presence information.
+
+[https://centrifugal.github.io/centrifugo/server/http_api/#presence_stats](https://centrifugal.github.io/centrifugo/server/http_api/#precence_stats)
 
 ```ruby
-
-	#routes.rb or any router lib
-	post 'sockets/auth'
-
-	# Auth method to authenticate private channels
-	#sockets_controller.rb or anywhere in your app
-
-	# client = Instance of Centrifuge initialized // check Usage section
-	# params[:channels] = single/array of channels
-	def auth
-	  if user_signed_in? # Or use a before_filter
-	    data = {}
-	    sign = client.generate_channel_sign(
-	        params[:client], params[:channels], "{}"
-	    )
-	    data[channel] = {
-	        "sign": sign,
-	        "info": "{}"
-	    }
-	    render :json => data
-	  else
-	    render :text => "Not authorized", :status => '403'
-	  end
-	end
+client.presence_stats(channel: 'chat')
+
+# {
+#   "result" => {
+#     "num_clients" => 0,
+#     "num_users" => 0
+#   }
+# }
 ```
-On client side initialize Centrifuge object like so:
 
-``` javascript
+#### History
 
-	// client_info = valid JSON string of user_info
-	// client_token = client.token_for (described above)
+Get channel history information (list of last messages published into channel).
 
-	var centrifuge = new Centrifuge({
-	   url: "http://localhost:8000/connection",
-	   user: window.currentUser.id,
-	   timestamp: window.currentUser.current_timestamp,
-	   debug: true,
-	   info: JSON.stringify(window.client_info),
-	   token: window.client_token,
-	   refreshEndpoint: "/sockets/refresh",
-	   authEndpoint: "/sockets/auth",
-	   authHeaders: {
-	      'X-Transaction': 'WebSocket Auth',
-	      'X-CSRF-Token': window.currentUser.form_authenticity_token
-	   },
-	   refreshHeaders: {
-	      'X-Transaction': 'WebSocket Auth',
-	      'X-CSRF-Token': window.currentUser.form_authenticity_token
-	   }
-	 });
+[https://centrifugal.github.io/centrifugo/server/http_api/#history](https://centrifugal.github.io/centrifugo/server/http_api/#hisotry)
 
-	centrifuge.connect();
-
-	// If you using jbuiler use raw render(template_name) and use a global window object to load the user information.
+```ruby
+client.history('chat') 
```

Comment:
Centrifugo v3 will change semantics of history call a bit, users will need to provide an explicit limit, by default only top stream position offset and current stream epoch will be returned, but I suppose we can address this later. 

Also, I noticed that this is the only API method where positional function argument is used, I think we should be consistent and use `client.history(channel: 'chat')` here (maybe it's already this way in function declaration).